build.rs

```diff
@@ -0,0 +1,19 @@
+fn main() {
+    // Docs.rs only target is x86_64
+    // `include/simsimd/types.h:119:9: error: _Float16 is not supported on this target`
```

Comment:
We can define a `SIMSIMD_NATIVE_F16=0` macro, that will avoid using this type and will make compilation pass with most compilers.

Reply:
Nice, but hard-coded in the `build.rs` or include conditionally?